test_suite/cloud/test_aws.py

```diff
@@ -558,8 +558,27 @@ def test_yum_group_install(self, host):
             pytest.skip('Not applicable to Atomic host AMIs')
 
         with host.sudo():
-            assert host.run_test('yum -y groupinstall "Development tools"'), \
-                'Error while installing Development tools group'
+            dev_tools_install_command = 'yum -y groupinstall "Development tools"'
+            result = test_lib.print_host_command_output(host, dev_tools_install_command, capture_result=True)
+
+            if result.failed:
+                err_message = "This system is not registered to Red Hat Subscription Management"
+                if err_message in result.stderr:
+                    print('"Development tools" installation attempt failed. Trying to apply a workaround...')
+                    host.run(
+                        'echo -e "enabled=0" > /etc/yum/pluginconf.d/subscription-manager.conf'
+                        ' && yum clean all'
+                    )
+
+            result_second_attempt = test_lib.print_host_command_output(
+                host, dev_tools_install_command, capture_result=True
+            )
+
+            assert result_second_attempt.succeeded, (
```

Comment:
This should be indented inside the if statement. Maybe it was a mistake from my code suggestion.